### PR TITLE
A post view

### PR DESCRIPTION
**Pull request by a-sh-dev**
- WIP for HOME that involves POSTS and POST
- add local postsData for protoyping
- add images as assets for about page and test post content

### CODE REVIEW

**Review by a-sh-dev — COMMENTED** [review]
need to fix something first...

**Comment by a-sh-dev** [comment]
HOME, POSTS and POST are still working in progress...

File: src/components/posts/Posts.js [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
+            <Post post={post} />
+          </div>
+        );
+      })}
+
+      <hr />
+      <small>test with local fake data</small>
+      {data.map((post) => {
+        return <p key={post._id}>{post.title}</p>;
+      })}
```
Comment (NatachaBartu) [inline src/components/posts/Posts.js:45]: nice logic for display the post! nice and clear!

File: src/data/postsData.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// Posts data for development - schema taken and modified from the server
```
Comment (NatachaBartu) [inline src/data/postsData.js:1]: Nice Dummy Data for Posts!Wel done Angie

**Review by NatachaBartu — COMMENTED** [review]
Welll done Angie! A lot of work!

**Pull request merged**